Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/JointLimitRotationManipulators.cpp

```diff
@@ -156,4 +136,42 @@ namespace EMotionFX
             ? m_physicsSetupManipulatorData.m_jointConfiguration->m_parentLocalRotation
             : m_physicsSetupManipulatorData.m_jointConfiguration->m_childLocalRotation;
     }
+
+    void JointLimitRotationManipulators::InvalidateEditorValues()
+    {
+        if (m_physicsSetupManipulatorData.m_jointLimitWidget)
+        {
+            m_physicsSetupManipulatorData.m_jointLimitWidget->InvalidateValues();
+        }
+    }
+
+    void CreateCommandAdjustJointLimit(MCore::CommandGroup& commandGroup, const PhysicsSetupManipulatorData& physicsSetupManipulatorData)
+    {
+        if (!commandGroup.IsEmpty())
+        {
+            return;
+        }
+        commandGroup.SetGroupName("Adjust joint limit");
+        const AZ::u32 actorId = physicsSetupManipulatorData.m_actor->GetID();
+        const AZStd::string& nodeName = physicsSetupManipulatorData.m_node->GetNameString();
+        CommandAdjustJointLimit* command = aznew CommandAdjustJointLimit(actorId, nodeName);
+        commandGroup.AddCommand(command);
+        command->SetOldJointConfiguration(physicsSetupManipulatorData.m_jointConfiguration);
+    }
+
+    void ExecuteCommandAdjustJointLimit(MCore::CommandGroup& commandGroup, const PhysicsSetupManipulatorData& physicsSetupManipulatorData)
+    {
+        if (commandGroup.IsEmpty())
+        {
+            return;
+        }
+
+        if (CommandAdjustJointLimit* command = azdynamic_cast<CommandAdjustJointLimit*>(commandGroup.GetCommand(0)))
```

Comment:
nit: Since there is a cast indicating the type you could make the line shorter by using `auto* command...`

Reply:
OK, will update.